src/screens/MemberDetail/MemberDetail.tsx

```
@@ -57,207 +48,138 @@
-      setFormState((prevState) => ({
-        ...prevState,
-        image: base64 as string,
-      }));
-    } else {
-      setFormState((prevState) => ({
-        ...prevState,
-        [name]: value,
-      }));
+  const handleUserUpdate = async (): Promise<void> => {
+    // Remove empty fields from the form state
+    function removeEmptyFields(obj: typeof formState) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)